Core/src/Batch/BatchTrait.php

```diff
@@ -94,7 +94,7 @@ public function send(array $items)
             if ($this->debugOutput) {
                 fwrite(
                     $this->debugOutputResource ?: STDERR,
-                    $e->getMessage() . PHP_EOL
+                    $e->getMessage() . PHP_EOL . $e->getTraceAsString()
```

Comment:
Since the old message was suffixed with a newline, the trace should be as well. In fact, two lines to make a visual separation between errors would be even better.

Reply:
Sure, done